docs/advanced/mbed_targets.md

```diff
@@ -18,15 +18,20 @@ mbed uses JSON as a description language for its build targets. The JSON descrip
         "detect_code": ["0230"]
 ```
 
-The definition of the target called **TEENSY3_1** is a JSON object. The properties in the object are either "standard" (understood by the mbed build system) or specific to the target.
+The style that we use for targets is:
+ - Each property is on its own line.
+ - The open brace `{` is on the same line as the target name, or the property name that is its key.
+ - The close brace `}` or close brace and comma `},` are on their own line.
+ - Lists take up at most one line.
+ - Long lines are not split.
 
 # Standard properties
 
-This section lists all the properties that are known to the mbed build system. Unless specified otherwise, all properties are optional.
+This section lists all the properties the mbed build system understands. Unless specified otherwise, all properties are optional.
 
-## inherits
+## `inherits`
 
-The description of an mbed target can "inherit" from one or more descriptions of other targets. When a target **A** inherits from another target **B**, (**A** is the _child_ of **B** and **B** is the _parent_ of **A**), it automatically "borrows" all the definitions of properties from **B** and can modify them as needed (if you're familiar with Python, this is very similar to class inheritance). In our example above, `TEENSY3_1` inherits from `Target` (most mbed targets inherit from `Target`). This is how `Target` is defined:
+The description of an mbed target can "inherit" from one or more descriptions of other targets. When a target, called a _child_ inherits from another target, called its _parent_, the child automatically copies all the properties from the parent. After the child has copied the properties of the parent, it may then overwrite, add or remove those properties. In our example above, `TEENSY3_1` inherits from `Target`. This is the definition of `Target`:
```

Comment:
Probably keep the from that you removed. you cannot add or remove the property as a whole when it's inherited, you can add or remove members of the property's list though.